example/tests/unit/test_pagination.py

```diff
@@ -76,3 +77,22 @@ def test_valid_offset_limit(self):
 
         assert queryset == list(range(offset + 1, next_offset + 1))
         assert content == expected_content
+
+    def test_LimitOffset_deprecation(self):
+        with pytest.warns(DeprecationWarning) as record:
+            pagination.LimitOffsetPagination()
+        assert len(record) == 1
+        assert 'LimitOffsetPagination' in str(record[0].message)
+
+
+# TODO: This test fails under py27 but it's not clear why so just leave it out for now.
```

Comment:
There is a pytest feature called xfail. See https://docs.pytest.org/en/latest/skipping.html

So instead of commenting this out you can mark it as `@pytest.mark.xfail` and still leave you comment with `TODO`.

This is that `CI` doesn't fail even when test is not successful but it still counts towards coverage.

Reply:
done. Got caught by flake8 again (#436 will address this if and when done)